robotis-op2/bibliography.md

```diff
@@ -0,0 +1,17 @@
+# Bibliography
+
+For any information about Webots, please visit [Cyberbotics
+website](http://www.cyberbotics.com).
+
+For any information about the Robotis-OP 2 robot, please visit [ROBOTIS
```

Comment:
Ok, so here it is wrong because of the '-'.

But personally I think it is better to use the correct spelling.
Note that also the book title doesn't contain the correct spelling of the robot name.